elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -264,7 +258,15 @@ class PfeTabs extends PFElement {
     this._selectTab(tab);
   }
 
-  _onSlotChange() {
+  _init() {
+    if (!this.hasAttribute("role")) {
+      this.setAttribute("role", "tablist");
```

Comment:
Maybe force this to be tablist instead of allowing custom?

Reply:
Fixed!